### ISSUE TITLE
context menu entries still present after uninstalling

### ISSUE THREAD

**Issue by maxcacheleux** [mintty/wsltty#75]
Context menu entries are still here even though I ran the uninstall script (remove from context menu) several times.
Running the script as administrator doesn't fix it.
Reboot doesn't fix it neither.

I'm running very last microsoft update (fall creator) with wsltty 1.8.1

**Comment by mintty** [comment]
Please open a Windows console (Win+R cmd), enter the following commands and quote the output here:
```
cd %LOCALAPPDATA%\wsltty
bin\dash config-distros.sh -contextmenu-remove
```

**Comment by maxcacheleux** [comment]
Doing
```
cd %LOCALAPPDATA%\wsltty
bin\dash config-distros.sh -contextmenu-remove
```
in cmd.exe results in no output at all and just gives me the prompt back:
```
C:\Users\max>cd %LOCALAPPDATA%\wsltty

C:\Users\max\AppData\Local\wsltty>bin\dash config-distros.sh -contextmenu-remove

C:\Users\max\AppData\Local\wsltty>
```
Context menu entries are still present too.

**Comment by mintty** [comment]
Run this before calling the script as above:
`PATH /bin;%PATH%`

To be fixed in script.

**Comment by maxcacheleux** [comment]
Fixed. Context menu entries are now correctly removed.

Thanks for your help!

**Comment by mintty** [comment]
Released 1.8.2.
